src/common/utils/auth/getUserOrServerRedirect.ts

```diff
@@ -1,20 +1,30 @@
-import { createClient } from '@/common/services/auth/supabase/server'
+import { createClient } from '@/features/auth/supabase/server'
```

Comment:
Is `createClient` only for `auth`? Isn't it more broad for database, etc?

Reply:
yeah good point, moved to `common/data/supabase`